examples/http-external/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+const resourceGroup = new azure.core.ResourceGroup('example', { location: 'West US 2' });
```

Comment:
perhaps the location should come from config?  i don't care much though.

Reply:
I think it's hard-coded in all other samples in this repo